services/app-api/src/parameterStore/getStateAnalystEmailsStore.ts

```diff
@@ -0,0 +1,25 @@
+import { Context } from '../handlers/apollo_gql'
+import { logError } from '../logger'
+import { setErrorAttributesOnActiveSpan } from '../resolvers/attributeHelper'
+import { getParameterStore } from './parameterStore'
+
+export const getStateAnalystEmailsStore = async (
```

Comment:
nit: I’m not sure “store” is adding anything to this function name. Seems like since we pass in the state code we are just getting a list of analysts out, so I’d name this function “getStateAnalystEmails” and leave the package to have “store” in the name.